packages/@sanity/imagetool/src/HotspotImage.js

```diff
@@ -39,6 +40,7 @@ export default class HotspotImage extends React.PureComponent {
   static defaultProps = {
     alignX: 'center',
     alignY: 'center',
+    className: undefined,
```

Comment:
No need to specify undefined as default prop

Reply:
I get a warning from eslint if I don't.